packages/Pill/src/Pill.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import PropTypes from "prop-types";
+import tokens from "@paprika/tokens";
+import RawButton from "@paprika/raw-button";
+import pillStyles from "./Pill.styles";
+
+export const tokenPillColors = [
+  tokens.color.black,
+  tokens.color.blackLighten60,
+  tokens.color.blue,
+  tokens.color.greenDarken10,
+  tokens.color.orangeDarken10,
```

Comment:
I really never liked this part of the Pill API.  I like the `severityPillColors` options, they are nicely semantic.  I'd support generic colour options (like `black`, `grey`, `blue`, `green`, `red`) but only supporting select _token_ colours seems like the wrong level of abstraction.